cloud/amazon/iam_policy.py

```diff
@@ -64,9 +64,9 @@
 '''
 
 EXAMPLES = '''
-# Create and policy with the name of 'Admin' to the group 'administrators'
+# Create a policy with the name of 'Admin' to the group 'administrators'
```

Comment:
I think your description in the name of the task is better, even though this is an improvement over the previous version